code/datums/mutations/body.dm

```diff
@@ -95,6 +95,12 @@
 /datum/mutation/human/dwarfism/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
+	// SKYRAT EDIT BEGIN
+	if(owner.dna.features["body_size"] < 1)
+		to_chat(owner, "You feel your body shrinking even further, but your organs aren't! Uh oh!")
+		owner.gib(FALSE, FALSE, TRUE) // Don't try to become unclickable!
+		return
+	// SKYRAT EDIT END
```

Comment:
Look

I get that its poor balance with all this size changing

But **what the fuck**, you arent going to gib people over this.

Reply:
oh wow what the hell, yeah i didnt even notice the gibbing

Reply:
note that it drops the brain still(see first arg documentation on gib)

the only reason im removing this because lmao fucking dna dart gun exists

Reply:
@Xymec it's not RR, once again, it drops the brain, you're 100% revivable